src/ruby.rs

```diff
@@ -222,10 +222,12 @@ impl zed::Extension for RubyExtension {
             }
         }
 
-        if !arguments.contains(&"--command".to_string()) {
-            // Ensure that all arguments are passed after a "--", as required by rdbg.
-            arguments.push("--".into());
+        if let Some(configuration) = configuration.as_object_mut() {
```

Comment:
I think we need to rebase this PR to pull the latest changes. Would you mind checking it, please?